app/javascript/components/server-components/Settings/PaymentsPage.tsx

```diff
@@ -888,9 +889,27 @@ const PaymentsPage = (props: Props) => {
                 options={PAYOUT_FREQUENCIES.map((frequency) => ({
                   id: frequency,
                   label: frequency.charAt(0).toUpperCase() + frequency.slice(1),
+                  disabled: frequency === "daily" && !props.payout_frequency_daily_supported,
                 }))}
               />
+              <small>
+                Daily payouts are only available for US users with eligible bank accounts and more than 4 previous
+                payouts.
+              </small>
             </fieldset>
+            {payoutFrequency === "daily" && props.payout_frequency_daily_supported ? (
+              <div role="status" className="info">
+                <div>
+                  Every day, your balance from the previous day will be sent to you via instant payouts, subject to a{" "}
+                  <b>3% fee</b>.
+                </div>
+              </div>
+            ) : null}
+            {payoutFrequency === "daily" && !props.payout_frequency_daily_supported && (
```

Comment:
Technically a user can stop supporting instant payouts (even though their frequency preference is still saved as daily).

In this case, we show this warning in the UI. On the backend, the user will be skipped during daily instant payout job and will be paid out by the weekly payout job (as a fallback).